### DISCUSSION TITLE
How to prevent the "Differential bundling"?

### DISCUSSION THREAD

**Discussion by Taxi4you** [parcel-bundler/parcel#9053] [General]
As described here:
https://parceljs.org/features/production/#differential-bundling

I want to disable this. I only want one bundle that can be used by any browser since I got many issues with this in Safari.

**Comment by avoguga** [comment]
See this bro:

https://parceljs.org/languages/javascript/#browser-compatibility
https://parceljs.org/features/targets/#differential-bundling
https://parceljs.org/features/targets/#package.json%23browserslist

And see the [browserslist docs](https://github.com/browserslist/browserslist#full-list) for know how to set this parameters.
